### DISCUSSION TITLE
How does this work?

### DISCUSSION THREAD

**Discussion by gerroon** [buggyj/tiddlyclip#80] [Q&A] [answered]
Hi

I installed both the plugin and the extension but I cant get it to work. My TW is running on another pc on the same network. Do I need to setup different port or forward one?

thanks

**Comment by buggyj** [comment, marked as answer]
HI Gerroon,

The browser addon works with tiddlywikis that are loaded in the browser, there is no network interaction, clips are passed dirrectly from the addon to the tiddlywiki. 

There are some (somewhat dated) getting started instructions here:
http://tiddlyclip.tiddlyspot.com/#Quick%20Start%20and%20Basics

**Reply by buggyj** [reply]
It does not matter where your tiddlywiki is hosted, if you can load it into the browser where the addon is installed then it will be able to work with it.

**Comment by gerroon** [comment]
Thanks, that is much better, this should be mentioned somewhere.

The main issue I have with thisa tm is that right click snap creates two entroes in TW.

**Reply by buggyj** [reply]
You could edit the tiddler `defaultSnap` and remove the last row in the table.

**Reply by gerroon** [reply]
> You could edit the tiddler `defaultSnap` and remove the last row in the table.

Thanks for the reply. Where is it and how do i edit it?

**Reply by buggyj** [reply]
Click on the tiddlyclip icon, it appears at the top right of the page, 

![image](https://user-images.githubusercontent.com/2855795/212460396-f1ae43c3-828f-4bed-ab75-09bed0d10f02.png)


this will show the TiddlyClipConfig tiddler, and in there you will see ~defaultSnap as a link within the table. Clicking on the link will show that tiddler. Then click the edit button at the top left of that tiddler.

![image](https://user-images.githubusercontent.com/2855795/212460456-805eb7cb-5e7c-4172-ae51-3321983d1018.png)

**Reply by gerroon** [reply]
> Click on the tiddlyclip icon, it appears at the top right of the page,
> 
> ![image](https://user-images.githubusercontent.com/2855795/212460396-f1ae43c3-828f-4bed-ab75-09bed0d10f02.png)
> 
> this will show the TiddlyClipConfig tiddler, and in there you will see ~defaultSnap as a link within the table. Clicking on the link will show that tiddler. Then click the edit button at the top left of that tiddler.
> 
> ![image](https://user-images.githubusercontent.com/2855795/212460456-805eb7cb-5e7c-4172-ae51-3321983d1018.png)

Thank you, I think it works better now.

**Comment by gerroon** [comment]
Also is it possible to snip with better formtting? 

I was testing it with this page and all the text comes in totally useless format, I am wondering if it is possible to improve the snipped text layout.  I tried select some text of couple paragraphs then right click ->snip

**Reply by arunnbabu81** [reply]
https://github.com/buggyj/tiddlyclip-plugin/discussions - Check the discussion section of tiddlyclip plug in to get more ideas. 

Also [here](https://demo-complete.tiddlyhost.com/#TiddlyClipConfig) is a list of Tiddlyclipcongfig I have created using buggy's help. The list is a little big and complicated to understand since I haven't documented it much. But since it is sorted by different categories you may get some idea. Ask if you need some help.

**Reply by buggyj** [reply]
The default is to copy plain text, but this can be changed to html in the `defaultSnip` tiddler:

![image](https://user-images.githubusercontent.com/2855795/212466828-639f1fff-47ca-4bc2-b196-fb11f27536fb.png)

changing `((*@text*))`  to `((*@web*))`

The contains of the clipboard are also available via

`((*@clip*))`

**Reply by gerroon** [reply]
Wonderful replies, thank you both. This is great stuff.
